docs/install/yaml-install/eventing/install-eventing-with-yaml.md

```diff
@@ -50,8 +50,28 @@ Follow the procedure for the Channel of your choice:
 
 === "Apache Kafka Channel"
 
-    1. Install [Strimzi](https://strimzi.io/quickstarts/).
-    1. Install the Apache Kafka Channel for Knative from the [`knative-sandbox` repository](https://github.com/knative-sandbox/eventing-kafka).
+
+    The following commands install the KafkaChannel and run event routing in a system
+    namespace. The `knative-eventing` namespace is used by default.
+
+    1. Install the Kafka controller by running the following command:
+
+        ```bash
+        kubectl apply -f {{ artifact(org="knative-sandbox",repo="eventing-kafka-broker",file="eventing-kafka-controller.yaml")}}
```

Comment:
do we need to mention how to disable the other controllers (e.g. for the source)?